DEVELOPING.md

```diff
@@ -1,4 +1,32 @@
-This project is a monorepo. The different components are:
+# Developing
+If you are new to contributing, please start here. This guide will cover setting up for any of our apps and should be read before delving into the specifics of each app.
+
+## Table of contents
+- [Before you start](#Before-you-start)
+- [Directory structure](#Directory-structure)
+- [Environments](#Environments)
+- [Set up environment variables](#Set-up-environment-variables)
+- [Managing Node.js versions](#Managing-Node.js-versions)
+- [Mock data](#Mock-data)
+- [Running apps](#Running-apps)
+- [Linting](#Linting)
+- [Testing](#Testing)
+- [Submitting your contributions](#Submitting-your-contributions)
+- [Deploying](#Deploying)
+
+## Before you start
```

Comment:
previously this was done ad-hoc when we have new contributors and not all perms were given. Hopefully this will reduce that.